src/tests/adr/vla_adr.rs

```diff
@@ -107,7 +107,17 @@ fn representation() {
         },
         nature: Any,
         location: SourceLocation {
-            span: None,
```

Comment:
It's interesting that we now get a source location for the internal array pointer in the vla-struct. Not an issue per-se I think, just interesting 😃